t/01_run.t

```diff
@@ -540,4 +540,24 @@ process';
 'setting MOJO_PROCESS_DEBUG to 1 enables debug mode when executing external process';
 };
 
+subtest 'process_args' => sub {
+  use Mojo::IOLoop::ReadWriteProcess;
```

Comment:
Having this in each and every subtest is useless. I would be better to just use it once at the top.

Reply:
yeah I just followed the style of the other tests. I would say cleaning the tests is a other topic.

I also think that this external helpers should be in that repo: https://github.com/mudler/Mojo-IOLoop-ReadWriteProcess/blob/master/t/01_run.t#L131 

And I'm not sure about this line: https://github.com/mudler/Mojo-IOLoop-ReadWriteProcess/blob/master/t/01_run.t#L9 I guess this should be correctly set by the caller of `prove`.

Reply:
It mostly to ensure perl is able to find extra libs and other stuff at run time... all the magiv happens at `BEGIN {}` block